spec/javascript/components/supplemental_application/SupplementalApplicationPage.test.js

```diff
@@ -70,6 +75,23 @@ describe('SupplementalApplicationPage', () => {
     expect(tree).toMatchSnapshot()
   })
 
+  test('it saves expected values if no changes are made', async () => {
```

Comment:
added this because we didn't have an explicit test of what gets sent to the API on save